components/compass-runtime-agent/internal/kyma/applications/converters/gateway_for_ns.go

```diff
@@ -10,6 +10,15 @@ import (
 
```

Comment:
There are not used constants defined in util.go. Please remove them.

Reply:
Fixed